src/Tools/setup-snapshot.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+PCL_NAME=PortableReferenceAssemblies-2014-04-14
+
+if [ "$EUID" -ne 0 ]; then
+    echo "Error: This script must be run as root"
+    exit 1
+fi
+
+apt-get -y install mono-snapshot-latest
+
```

Comment:
@VSadov unfortunately it's hard to pick a fixed version because they cycle through packages so fast.  Open to suggestions on how to handle this better.  

This could be updated though to take a parameter to specify a specific snapshot though.